README.md

```diff
@@ -34,36 +34,38 @@ make check
 typedef struct mlog_data { /* implementation defined */ } mlog_data_t;
 ```
 
+Log buffering structure for a whole MassiveLogger environment.
+
 ### mlog_init
 ```c
 void mlog_init(mlog_data_t* md, int num_ranks, size_t buf_size);
 ```
 
 Parameters:
-* `md`        : Global log data for MassiveLogger.
-* `num_ranks` : The number of ranks (e.g., workers or threads) who use MassiveLogger.
-* `buf_size`  : Size of the begin buffer and the end buffer.
+* `md`        : Pointer to a `mlog_data_t` variable at global scope.
+* `num_ranks` : The number of ranks (e.g., the number of workers/threads using MassiveLogger).
+* `buf_size`  : Initial size of buffering memories in the `mlog_data_t` variable.
```

Comment:
The plural form "memories" sounds a bit strange for me. Maybe it is uncountable in computing.

By the way, we should have noted that it allocates
- two buffers that have `size` bytes of memory respectively
- per rank,
- so the total size of memory will be 2 * `size` * `num_ranks`.

Reply:
> The plural form "memories" sounds a bit strange for me. Maybe it is uncountable in computing.

That is certainly true. The "memories" word is not proper for explaining this kind of thing. I will change the expression of that.

I also agree to note what the function allocates internally.
However, using "begin buffer" and "end buffer" words at the beginning of README can confuse readers since they are not explained before.
I should take care of word choice for a reader-friendly explanation.

Reply:
You are absolutely right. The problem here is that we don't explain how it works before API specs, which should be an another issue.